_events/2019/supercomputing-2019.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Supercomputing 2019 (SC19)
+location: Denver, CO
+url: https://sc19.supercomputing.org/
+expires: 2019-11-22
```

Comment:
I think this is missing the date field too.

Reply:
There is no date field :) If you add one, it will break the events from rendering. The author needs to write the date up front in the content section (below this header). For example, when I implemented it with a data file (_data/events.yml) we didn't have a date field either, only an expires.